docs/HowToUsePeripheral.md

```diff
@@ -123,7 +135,8 @@ UART has two parts, setting early console and setting serial console driver, ser
 ### Setting of early console
 #### Add up_earlyserialinit function
 Boot code must call this before os starting, to provide low level debug messages during bootup.
```

Comment:
I think passive voice would be better, as the focus is on the API, not the entity calling it. So, something like "This function must be called during the boot routine, before starting os".